Lib/uuid.py

```diff
@@ -1,8 +1,9 @@
 r"""UUID objects (universally unique identifiers) according to RFC 4122.
 
 This module provides immutable UUID objects (class UUID) and the functions
-uuid1(), uuid3(), uuid4(), uuid5() for generating version 1, 3, 4, and 5
-UUIDs as specified in RFC 4122.
+uuid1(), uuid3(), uuid4(), uuid5() for generating version 1 to 8 UUIDs as
+specified in RFC 4122 (superseeded by RFC 9562 but will still be referred
+to as RFC 4122 in the future for compatibility purposes).
```

Comment:
```suggestion
uuid1(), uuid3(), uuid4(), uuid5(), uuid6(), uuid7() and uuid8() for
generating version 1 to 8 UUIDs as specified in RFC 4122 (superseeded
by RFC 9562 but will still be referred to as RFC 4122 in the future
for compatibility purposes).
```

Reply:
Oh good catch! thank you (I'll update the parenthesis as well)